addons/core/config.cpp

```diff
@@ -44,8 +44,8 @@ class CfgMods {
     class PREFIX {
         dir = "@task_force_radio";
         name = "Task Force Arrowhead Radio";
-        tooltip = "Task Force Arrowhead Radio";
-        overview = "Task Force Arrowhead Radio";
+ -      tooltip = "Task Force Arrowhead Radio";
+ -      overview = "Task Force Arrowhead Radio";
```

Comment:
why are you adding `-` ?

Reply:
Wow, I don't write this. Might editor problem, I will fix